### PR TITLE
#5

### PR DESCRIPTION
**Pull request by netmanito**
We need to review and add Alastria related data on Ether, GAS, and other definitions. That's all on this branch.

### CODE REVIEW

File: HELP.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+* Name and Company representing
+* e-mail address to be added.
+* channels list where to be joined.
+
+### Available channels
+
+* `#general` General purpose channel (public)
+* `#core` Core Team channel (private)
+* `#notificaciones` Notifications channel (public)
+* `#validadores` Validators administrators channel (permissioned, only for people who manages validator nodes)
```
Comment (marcosio) [inline HELP.md]: It is more appropriate `permission` instead of `permissioned`?

File: HELP.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+* `#develteam` Channel for the develteam Alastria
```
Comment (marcosio) [inline HELP.md]: It is more appropriate `devel team`instead of `develteam`?

File: HELP.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+* `#frontend-security` Frontend security group
+
+ 
+## Github (use & governance)
+
+How to colaborate or open issues on Github platform.
```
Comment (marcosio) [inline HELP.md]: `collaborate` instead of `colaborate`.

File: HELP.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+
+### Open an issue
+
+If you need to open an Issue on the alastria-node, please follow the instructions below.
+
+### Create a pull request
+
+If you want to do a pull request on alastria-node repository, please follow these instructions.
```
Comment (marcosio) [inline HELP.md]: `on the alastria-node` instead of `on alastria-node`.

File: generic_FAQ_EN.md [hunk 5]
```diff
@@ -0,0 +1,78 @@
+1. [What is alastria](generic_FAQ_EN.md#what-is-alastria)
+2. [What is Quorum](generic_FAQ_EN.md#what-is-quorum)
+3. [What is Ethereum](generic_FAQ_EN.md#what-is-ethereum)
+4.  [What is the difference between private, permissioned and public networks, What is the Alastria case?](generic_FAQ_EN.md#what-is-the-difference-between-private-permissioned-and-public-networks-what-is-the-alastria-case)
+5. [What is ETHER, how it affects to test-net 1](generic_FAQ_EN.md#what-is-ether-how-it-affects-to-test-net-1)
+6. [What is GAS, how it affects to test-net 1](generic_FAQ_EN.md#what-is-gas-how-it-affects-to-test-net-1)
+7. [The appearence of Solidity added](generic_FAQ_EN.md#the-appearance-of-solidity-added)
```
Comment (marcosio) [inline generic_FAQ_EN.md]: `appearance` instead of `appearence`.

File: generic_FAQ_EN.md [hunk 6]
```diff
@@ -0,0 +1,78 @@
...
+8. [Aspects of programming in Quorum](generic_FAQ_EN.md#aspects-of-programming-in-quorum)
+
+## What is alastria
```
Comment (marcosio) [inline generic_FAQ_EN.md]: `Alastria` instead of `alastria`.

File: generic_FAQ_EN.md [hunk 7]
```diff
@@ -0,0 +1,78 @@
...
+
+Alastria can be defined as a neutral, semipublic, independet, and permissioned Blockchain/DLT network, designed to work with current regulations, and let the associated companies to experiment with this technologies in a corporative environment.
```
Comment (marcosio) [inline generic_FAQ_EN.md]: `independent`instead of `independet`.

Comment (marcosio) [inline generic_FAQ_EN.md]: `companies experiment` instead of `companies to experiment`.

Comment (marcosio) [inline generic_FAQ_EN.md]: `these technologies` instead of `this technologies`.

File: generic_FAQ_EN.md [hunk 8]
```diff
@@ -0,0 +1,78 @@
...
+
+* **Governance**: Every enterprise needs to design standards, processes, methods, and tools to develop and operate a private blockchain. To achieve this they will need tools and frameworks such as IDE, testing framework, security auditing tool etc. For long-term successful operation, they also need to develop high-quality documentation. This requires proactive governance.
+
+[Reference](https://www.devteam.space/blog/public-vs-private-permissioned-blockchain-comparison/)
+
+
+## What is Ether, how it affects to test-net 1
+
+Ether is a necessary element — a fuel — for operating the distributed application platform Ethereum. It is a form of payment made by the clients of the platform to the machines executing the requested operations. To put it another way, ether is the incentive ensuring that developers write quality applications (wasteful code costs more), and that the network remains healthy (people are compensated for their contributed resources).
```
Comment (marcosio) [inline generic_FAQ_EN.md]: ` (wasteful code costs more), and that`. The comma appears to be unnecessary?

File: generic_FAQ_EN.md [hunk 9]
```diff
@@ -0,0 +1,78 @@
...
+-->
+
+[Refence](http://alastria.io/#1)
+
+## What is Quorum
+
+Enterprise-ready distributed ledger and smart contract platform. Quorum™ is an enterprise-focused version of Ethereum. Quorum is ideal for any application requiring high speed and high throughput processing of private transactions within a permissioned group of known participants.
+
+[Reference](https://www.jpmorgan.com/global/Quorum)
+
```
Comment (marcosio) [inline generic_FAQ_EN.md]: I think that this point may be completed with:
1. is the first approach to build the Alastria Network.
2. other teams can generate other test-nets with other clients or other consensus protocols.
3. we have full access to developer team of Quorum.

File: generic_FAQ_EN.md [hunk 10]
```diff
@@ -0,0 +1,78 @@
...
+* **Supportability**: Businesses need to ensure that their private blockchains are maintainable. This includes day-to-day administration, patching, and troubleshooting. They need to have processes in place to monitor service level agreements (SLAs) and manage operations.
+
+* **Integration**: Businesses have their systems of record (SOR) applications, which their enterprise blockchain will need to communicate with. Before they roll-out their blockchain, they need to have the required application programming interfaces (APIs) ready to implement this integration.
+
+* **Governance**: Every enterprise needs to design standards, processes, methods, and tools to develop and operate a private blockchain. To achieve this they will need tools and frameworks such as IDE, testing framework, security auditing tool etc. For long-term successful operation, they also need to develop high-quality documentation. This requires proactive governance.
+
+[Reference](https://www.devteam.space/blog/public-vs-private-permissioned-blockchain-comparison/)
+
```
Comment (marcosio) [inline generic_FAQ_EN.md]: I think that the public permissioned network is not explained yet. `Alastria is a public permissioned network`.

File: generic_FAQ_EN.md [hunk 11]
```diff
@@ -0,0 +1,78 @@
...
+
+## What is Ether, how it affects to test-net 1
+
+Ether is a necessary element — a fuel — for operating the distributed application platform Ethereum. It is a form of payment made by the clients of the platform to the machines executing the requested operations. To put it another way, ether is the incentive ensuring that developers write quality applications (wasteful code costs more), and that the network remains healthy (people are compensated for their contributed resources).
+
+[Reference](https://www.ethereum.org/ether)
```
Comment (marcosio) [inline generic_FAQ_EN.md]: In my opinion, at this point, we must explain that Quorum does not spend Ether in transaction processing by default and that we are working in a governance for it.

File: generic_FAQ_EN.md [hunk 12]
```diff
@@ -0,0 +1,78 @@
...
+
+## What is gas, how it affects to test-net 1
+
+**Gas (Ethereum)** refers to the pricing value required to successfully conduct a transaction or execute a contract on the Ethereum blockchain platform. 
+
+[Reference](https://www.investopedia.com/terms/g/gas-ethereum.asp)
+
```
Comment (marcosio) [inline generic_FAQ_EN.md]: In my opinion, at this point, we must explain that the amount of gas of a transaction must be sufficient for the transaction and how gasPrice=0 affect to the cost of the transaction to the account of the sender.

File: HELP.md (hunk 1 above)
Comment (netmanito) [reply HELP.md]: not sure how to do it, `permission needed`, la idea es poner publico, privado, con permiso

File: HELP.md (hunk 2 above)
Comment (netmanito) [reply HELP.md]: ok, that's fine

File: HELP.md (hunk 3 above)
Comment (netmanito) [reply HELP.md]: fixed

File: HELP.md (hunk 4 above)
Comment (netmanito) [reply HELP.md]: fixed

File: generic_FAQ_EN.md (hunk 5 above)
Comment (netmanito) [reply generic_FAQ_EN.md]: fixed

File: generic_FAQ_EN.md (hunk 6 above)
Comment (netmanito) [reply generic_FAQ_EN.md]: ok

File: generic_FAQ_EN.md (hunk 7 above)
Comment (netmanito) [reply generic_FAQ_EN.md]: ok

Comment (netmanito) [reply generic_FAQ_EN.md]: ok

File: generic_FAQ_EN.md (hunk 8 above)
Comment (netmanito) [reply generic_FAQ_EN.md]: ok, i think you're right

File: generic_FAQ_EN.md (hunk 9 above)
Comment (netmanito) [reply generic_FAQ_EN.md]: added

File: generic_FAQ_EN.md (hunk 7 above)
Comment (netmanito) [reply generic_FAQ_EN.md]: ok

File: generic_FAQ_EN.md (hunk 10 above)
Comment (netmanito) [reply generic_FAQ_EN.md]: what do you mean? That is defined at start in What is alastria, so I think it only should be a little comment. making again a definition is redundant.
I'll leave a breaf description and reference to the upper definition.

**Pull request merged**